Mathlib/Order/Bounds/Basic.lean

```diff
@@ -1374,35 +1374,34 @@ theorem mem_lowerBounds_image2 (ha : a ∈ lowerBounds s) (hb : b ∈ lowerBound
 #align mem_lower_bounds_image2 mem_lowerBounds_image2
 
 theorem image2_upperBounds_upperBounds_subset :
-    image2 f (upperBounds s) (upperBounds t) ⊆ upperBounds (image2 f s t) := by
-  rintro _ ⟨a, b, ha, hb, rfl⟩
-  exact mem_upperBounds_image2 h₀ h₁ ha hb
+    image2 f (upperBounds s) (upperBounds t) ⊆ upperBounds (image2 f s t) :=
+  image2_subset_iff.2 fun _ ha _ hb ↦ mem_upperBounds_image2 h₀ h₁ ha hb
```

Comment:
This maybe looks like weak evidence that we should fiddle with the argument order/explicitness so that we can write
```suggestion
  image2_subset_iff.2 <| mem_upperBounds_image2 h₀ h₁
```
but that's not a problem for this PR